generate_endf71.py

```diff
@@ -99,122 +100,200 @@ def sort_key(path):
+library_name = 'endf'
+release = 'b7.1'
+
+cwd = Path.cwd()
+
+wmp_files_dir = args.destination / 'neutron_wmp'
+endf_files_dir = cwd.joinpath('-'.join([library_name, release, 'endf']))
+neutron_dir = endf_files_dir / 'neutrons'
+thermal_dir = endf_files_dir / 'thermal_scatt'
+download_path = cwd.joinpath('-'.join([library_name, release, 'download']))
 
+temperatures = [250.0, 293.6, 600.0, 900.0, 1200.0, 2500.0]
+
+# This dictionary contains all the unique information about each release. This
+# can be extended to accommodate new releases
+release_details = {
+    'b7.1': {
+        'neutron': {
+            'base_url': 'http://www.nndc.bnl.gov/endf/b7.1/zips/',
+            'compressed_files': ['ENDF-B-VII.1-neutrons.zip',
+                                 'ENDF-B-VII.1-thermal_scatt.zip'],
+            'checksums': ['e5d7f441fc4c92893322c24d1725e29c',
+                          'fe590109dde63b2ec5dc228c7b8cab02'],
+            'file_type': 'endf',
+            'endf_files': endf_files_dir.rglob('n-*.endf'),
+            'sab_files': [
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinH2O.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinCH2.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-HinZrH.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-ortho-H.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-para-H.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-benzine.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-l-CH4.endf'),
+                (neutron_dir / 'n-001_H_001.endf', thermal_dir / 'tsl-s-CH4.endf'),
+                (neutron_dir / 'n-001_H_002.endf', thermal_dir / 'tsl-DinD2O.endf'),
+                (neutron_dir / 'n-001_H_002.endf', thermal_dir / 'tsl-ortho-D.endf'),
+                (neutron_dir / 'n-001_H_002.endf', thermal_dir / 'tsl-para-D.endf'),
+                (neutron_dir / 'n-004_Be_009.endf', thermal_dir / 'tsl-BeinBeO.endf'),
+                (neutron_dir / 'n-004_Be_009.endf', thermal_dir / 'tsl-Be-metal.endf'),
+                (neutron_dir / 'n-006_C_000.endf', thermal_dir / 'tsl-graphite.endf'),
+                (neutron_dir / 'n-008_O_016.endf', thermal_dir / 'tsl-OinBeO.endf'),
+                (neutron_dir / 'n-008_O_016.endf', thermal_dir / 'tsl-OinUO2.endf'),
+                (neutron_dir / 'n-013_Al_027.endf', thermal_dir / 'tsl-013_Al_027.endf'),
+                (neutron_dir / 'n-026_Fe_056.endf', thermal_dir / 'tsl-026_Fe_056.endf'),
+                (neutron_dir / 'n-014_Si_028.endf', thermal_dir / 'tsl-SiO2.endf'),
+                (neutron_dir / 'n-040_Zr_090.endf', thermal_dir / 'tsl-ZrinZrH.endf'),
+                (neutron_dir / 'n-092_U_238.endf', thermal_dir / 'tsl-UinUO2.endf')
+            ],
+            'compressed_file_size': 1,
+            'uncompressed_file_size': 2
```

Comment:
Not sure where these values are coming from. The incident neutron files are more than 200 MB:
```
-rw-rw-r-- 1 romano romano 226499549 Mar 17 07:10 ENDF-B-VII.1-neutrons.zip
-rw-rw-r-- 1 romano romano  10442639 Mar 17 07:10 ENDF-B-VII.1-thermal_scatt.zip
```

Reply:
added sizes of combined compressed and uncompressed neutrons (and thermal neutrons)